### DISCUSSION TITLE
"Calculate DC bias" result often unreadable

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#893] [Q&A]
The result of "Calculate DC bias" is often unreadable in many places, see the attached image.
The numerical strings are sometimes drawn over the symbols and wires, especially for horisontal wires.
Is there any way around this?
![DC bias readability](https://github.com/user-attachments/assets/767210ee-5c8b-4bd0-a6c5-589d48a8976a)

**Comment by ra3xdh** [comment]
This is a known problem. It presents since the Qt3 version of the application. There is no easy way to overcome this unless the DC bias show algorithm redesign. This workaround on user level is not possible. I don't promise the fix in the near future.

**Comment by ra3xdh** [comment]
You may try setting node labels on some interesting nodes and then perform "DC analysis" simulation (not DC sweep) with F2 and add a table on schematic.

**Comment by bnilsson11** [comment]
Good idea, I will try it.
